web/src/components/Settings/PreferencesSection.tsx

```diff
@@ -13,13 +15,17 @@ const PreferencesSection = () => {
   const userStore = useUserStore();
   const { appearance, locale } = globalStore.state;
   const { setting, localSetting } = userStore.state.user as User;
+  const [telegramUserId, setTelegramUserId] = useState<string>("");
   const visibilitySelectorItems = VISIBILITY_SELECTOR_ITEMS.map((item) => {
     return {
       value: item.value,
       text: t(`memo.visibility.${item.text.toLowerCase()}`),
     };
   });
 
+  useEffect(() => {
```

Comment:
Why need `useEffect`? 👀

Reply:
The input value need to cache in storage, so we can submit it after all input done.

`useEffect` to load value from store while loading or other case .

Reply:
🤔Can it be replaced like
```typescript
  useEffect(() => {
    globalStore.fetchSystemStatus();
  }, []);

  useEffect(() => {
    api.getSystemSetting().then(({ data: { data: systemSettings } }) => {
      const openAIConfigSetting = systemSettings.find((setting) => setting.name === "openai-config");
      if (openAIConfigSetting) {
        setOpenAIConfig(JSON.parse(openAIConfigSetting.value));
      }

      const telegramRobotSetting = systemSettings.find((setting) => setting.name === "telegram-robot-token");
      if (telegramRobotSetting) {
        setTelegramRobotToken(telegramRobotSetting.value);
      }
    });
  }, []);
```